content/blog/2024-06-12-rustl-2024-unconference.md

```diff
@@ -0,0 +1,200 @@
+Most were happy to see winit becoming more trait-oriented, and to see the `dpi` crate spun off from winit. 
+
+Glazier is still shelved and we're planning to port its features to Winit until it reaches feature parity.
+
+A discussion that came up was the idea of splitting more "vocabulary types" from Winit into standalone crates.
+Vocabulary crates, eg crates that only define types and very little code, have a very low cost in memory and build times, while allowing projects to interop with eg Winit without actually adding a dependency on Winit.
+
+This was connected with the discussions about text editing: having a vocabulary crate that would list Winit's events and also Parley's cursor movements would allow for a very low-dependency text editing widget.
+
+### Accessibility
+
+*Note: This section is also my personal opinion and doesn't represent the Linebender project either.*
+
+The subject of Accessibility came up informally, but I'll claim it was underexplored.
+
+While the subject was on the agenda, by the time we reached it the Unconference had been running long enough that everyone was tired and ready to leave or get to the catering.
+This is an unfortunate pattern that I think is worth pointing out: if projects don't make a deliberate effort to prioritize discussions of accessibility, the subject will be left out.
+
+Matt Campbell, the maintainer of AccessKit, could not attend the Unconference.
+I'm concerned that, in his absence, other maintainers might have felt a dilution of responsibilty: since the accessibility guy wasn't there, nobody felt empowered to bring up accessibility.
+
+One point that did come up during the discussion is one I'd like to hammer in over the next few weeks, is that the Rust ecosystem should adopt a holistic view of accessibility.
+Accessibility isn't just about screen readers, and making a framework accessible doesn't stop at adding a dependency to AccessKit, though doing so is a great first step.
+
+Rik asked if there was a minimal set of the AccessKit elements that people could be expected to port their UI to.
+The AccessKit framework is rich, and it's not obvious at first glance how much is needed versus nice-to-have.
+The current reference in the Rust ecosystem is [the AccessKit implementation in eGUI](https://github.com/emilk/egui/pull/2294).
+
+### Requests for Rust features
+
+Jon Kelley, the creator of Dioxus, had [a laundry list of features](https://dioxus.notion.site/Dioxus-Labs-High-level-Rust-5fe1f1c9c8334815ad488410d948f05e) he wanted from the Rust language.
+That list was later [filed in the Project Goals repository]((https://github.com/rust-lang/rust-project-goals/pull/10)).
+
+The requests were well-documented, with motivating examples in existing Dioxus code that would clearly be improved by the features.
+Most of them felt both self-contained and like they would tremendously help beginners working on their first Rust project.
```

Comment:
There were plenty of veterans at the unconference saying that the suggestions would help them personally, so I think it's misleading to paint these as beginner focused.

```suggestion
Most of them felt both self-contained and like they would tremendously help both beginners working on their first Rust project and long time veterans alike.
```